src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -408,7 +446,16 @@ private function getSubresourceResponse(bool $v3, $mimeTypes, bool $collection,
             ];
 
             if ($v3) {
-                $okResponse['content'] = array_fill_keys($mimeTypes, ['schema' => ['$ref' => sprintf('#/components/schemas/%s', $definitionKey)]]);
+                foreach ($mimeTypes as $mimeType) {
```

Comment:
Same here. Try to reduce the diff size as much as possible to ease the review process